.evergreen/buildvariants-and-tasks.in.yml

```diff
@@ -53,31 +57,31 @@ const PACKAGE_BUILD_VARIANTS = [
 
 const TEST_PACKAGED_APP_BUILD_VARIANTS = [
   {
-    name: 'test-packaged-app-ubuntu',
+    name: 'test-server-ubuntu',
```

Comment:
Those runs look weird in CI, only 2 tests are running in every variant:

<img width="1171" alt="image" src="https://github.com/mongodb-js/compass/assets/5036933/4f9cd6e6-11b8-4963-920f-132977642dac">

Reply:
that is weird yeah